dw-form-field.js

```diff
@@ -75,34 +72,59 @@ export class DwFormField extends Formfield {
       /**
        * Aligns label at top.
        */
-       alignTop: { type: Boolean, reflect: true, attribute: 'align-top' },
-      
+      alignTop: { type: Boolean, reflect: true, attribute: "align-top" },
+
       /**
        *  True if label is available
        */
       _hasLabel: {
         type: Boolean,
-        reflect: true
-      }
-    }
+        reflect: true,
```

Comment:
this and following 2 lines changes seems extra, please avoid such extra changes.